resource/composed/composed.go

```diff
@@ -30,10 +30,13 @@ import (
 	"github.com/crossplane/function-sdk-go/errors"
 )
 
+// I don't love the package-scoped state here but this seems like the
```

Comment:
want to add a `(negz)` mention so folks can see quickly who I is? 😇

Reply:
Oh whoops I was too quick on the draw and pushed the commit that addresses this to https://github.com/crossplane/function-sdk-go/pull/212.